src/DateTimePicker/Formatter.elm

```diff
@@ -56,15 +56,15 @@ timeFormatter dateTime =
     let
         ( hourString, amPm ) =
             if dateTime.hour == 12 then
-                ( "12", "pm" )
+                ( "12", "PM" )
```

Comment:
we were formatting these lowercase because we follow Chicago style on the monolith. (or at least this was the case last time I fixed a bug around dates). Best form there: "a.m.", but "am" is also acceptable.

Did it break the tests to have it lowercase?

Reply:
It did, but we can change the tests! Let's go with `"am"` because the formatter in this package is writing content to an input and I think teachers are more likely to want to type without the periods